src/opm/parser/eclipse/share/keywords/900_OPM/W/WMICP

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "WMICP",
+  "sections": [
+    "SPECIAL",
+    "SCHEDULE"
+  ],
+  "items": [
+    {
+      "name": "WELL",
+      "value_type": "STRING"
+    },
+    {
+      "name": "MICROBIAL_CONCENTRATION",
+      "value_type": "UDA",
```

Comment:
Do you *really* mean to use an `UDA` here? Maybe a good old `double` would be good enough?